docs/notebooks/data_transformation.pct.py

```diff
@@ -244,6 +255,71 @@ def normalise(x, mean=None, std=None):
 print(f"observation: {observations[arg_min_idx, :]}")
 
 
+# %% [markdown]
+# # Data transformation with the DataTransformModelWrapper class
+#
+# Actually, we can achieve the same effect with a standard `BayesianOptimizer` by using a `DataTransformModelWrapper`.
+#
+# First, we create a new model class that inherits from the model wrapper class and the kind of `TrainableProbabilisticModel` that we want to use.
+
+# %%
+from trieste.models.normalization import DataTransformModelWrapper
+
+class GPRwithDataNormalization(DataTransformModelWrapper, GaussianProcessRegression):
+    pass
+
+
+# %% [markdown]
+#
+# Now we create `DataTransformer`s for query_points and observations, and pass these into our new model class when we construct it.
```

Comment:
for the sake of illustration, we could use max-min transformation on query points and standardisation on observations